ReportProject/src/ReportPro.java

```diff
@@ -0,0 +1,119 @@
+import java.util.*;
+
+public class ReportPro {
+
+
+
+
+    public static void main(String [] args) {
+
+        double[] list = new double [7];
+        double max = list[0];
+        double min = Double.MAX_VALUE;
+        double avg = 0.0;
+        double sum = 0.0;
+        boolean isRunning = true;
+        int entry = 0;
+
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Welcome to the Report Builder");
+        System.out.println("===================================================");
+        System.out.println("Please enter your first name: ");
+        String first_name = sc.nextLine(); //input from user of the first name
+        String filterF = first_name.replaceAll("[^a-zA-Z]+", ""); //filters the name
+        System.out.println("Please enter your last name: ");
+        String last_name = sc.nextLine(); //input from user of the last name
```

Comment:
Read in the last name one character at a time. What is the terminating condition, what is the construct? does a for loop work when you don't know how long their last name is?